src/test/java/org/checkstyle/autofix/InputClassRenamer.java

```diff
@@ -51,7 +51,7 @@ private static final class ClassRenameVisitor extends JavaIsoVisitor<ExecutionCo
 
         @Override
         public J.ClassDeclaration visitClassDeclaration(J.ClassDeclaration classDecl,
-                                                        ExecutionContext executionContext) {
+                                                        ExecutionContext ctx) {
```

Comment:
we try to be less cryptic  and we see that being verbose is more beneficial in projects with hundreds of contributors (majority are junior level).

What is rationale to this rename ?

Reply:
The rationale here is consistency with the other recipes we have; it's such a common argument that within the context of a recipe `ctx` is always the `ExecutionContext`. You can of course comment out that particular rule if you feel strongly about it.